kclvm/cmd/src/util.rs

```diff
@@ -10,6 +11,28 @@ pub(crate) fn strings_from_matches(matches: &ArgMatches, key: &str) -> Option<Ve
     })
 }
 
+#[inline]
+pub(crate) fn hashmaps_from_matches(
```

Comment:
Failure to handle abnormal situations. For example:
https://github.com/KusionStack/KCLVM/blob/main/kclvm/driver/src/arguments.rs#L7

Reply:
fixed.